mutiny-core/src/node.rs

```diff
@@ -1459,6 +1459,7 @@ impl<S: MutinyStorage> Node<S> {
 pub(crate) fn scoring_params() -> ProbabilisticScoringFeeParameters {
     ProbabilisticScoringFeeParameters {
         base_penalty_amount_multiplier_msat: 8192 * 4, // default * 4
```

Comment:
maybe bump this up to 5....
```suggestion
        base_penalty_amount_multiplier_msat: 8192 * 5, // default * 5
```